supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -117,9 +134,9 @@ See xref:container[container] in the _General Conventions_ part.
 *See also*:
 
 [discrete]
-[[deployment]]
-==== deployment (noun)
-*Description*: A statement of intent by a user to roll out a new version of a configuration. A replication controller can be used for that and other purposes. All deployments are replication controllers, but not all replication controllers are deployments. To avoid confusion, do not refer to an overall OpenShift installation, instance, or cluster as an "OpenShift deployment".
+[[custom-resource]]
+==== custom resource (noun)
+*Description*: A custom resource (CR) is a resource implemented through the Kubernetes `CustomResourceDefinition` API. A custom resource is distinct from the built-in Kubernetes resources, such as the pod and service resources. Every CR is part of an API group.
```

Comment:
I wonder if "distinct from" is too strong, making it sound like they behave differently or are somehow managed more specially than the built-in resources. Maybe to add a bit more context as to why they're distinct, if sticking with that wording:

"While custom resources behave the same as the built-in set of Kubernetes and OpenShift resources, such as the pod and service resources, custom resources are distinct in that they are added to the cluster by administrators either manually or by installing Operators, and therefore may not be available on all clusters by default.

That's a long sentence but you get the idea. Also added "OpenShift resources" in there so that one doesn't assume that we consider the default OpenShift-provided ones as "custom resources" either (I'm not actually sure whether technically they are, though).

Reply:
Thanks, I've replaced that sentence with your suggestion.